one_fm/utils.py

```diff
@@ -3640,4 +3640,38 @@ def send_work_anniversary_reminders():
                     others = [d for d in anniversary_persons if d != person]
                     reminder_text = get_work_anniversary_reminder_text(others)
                     send_work_anniversary_reminder(person_email, reminder_text, others, message, sender)
-                    
\ No newline at end of file
+
+
+def set_employee_status_to_vacation():
+    # Get today's date
+    current_date = today()
+
+    # Fetch approved leave applications where `from_date` is today or earlier, along with the employee's status
+    leave_applications = frappe.get_all('Leave Application', 
+        filters={
+            'status': 'Approved',
+            'from_date': ['<=', current_date],
+            'to_date': ['>', current_date]
+        },
+        fields=['employee', 'from_date', 'employee.status']  # Fetch employee status directly
```

Comment:
@yusuffAFEX , you are collecting , the "from_date" field, but you are not using it, also, if that employee.status, would work, also you are not using the field

Reply:
Okay